internal/pkg/template/input/options.go

```diff
@@ -13,13 +13,13 @@ import (
 type Options []Option
 
 type Option struct {
-	Id   string `json:"id" validate:"required"`
-	Name string `json:"name" validate:"required"`
+	Value string `json:"id" validate:"required"`
+	Label string `json:"name" validate:"required"`
```

Comment:
Nezmenil si to v JSON(net), preto nespadol ani ziaden test.

Reply:
Aha 😅

Reply:
Tak fixunuto https://github.com/keboola/keboola-as-code/pull/613/commits/6f3580b6eab61587b856398d42ac187074da2b30 🙂